op-batcher/batcher/throttle_controller_test.go

```diff
@@ -0,0 +1,769 @@
+package batcher
+
+import (
+	"math"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-batcher/config"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// noopLogger creates a logger that discards all output for testing
+func noopLogger() log.Logger {
+	return log.Root()
+}
+
+// TestStepController tests the step controller behavior
+func TestStepController(t *testing.T) {
+	strategy := NewStepStrategy(1000000, 5000, 21000, 130000)
```

Comment:
Be great to break out these numbers into named constants, in particular I want to be able to see what the threshold is set to (without diving in to the function signature).